pkg/resourcecreator/testdata/naisjob/vanilla_job_on_premises.yaml

```diff
@@ -123,7 +123,7 @@ tests:
                       capabilities:
                         drop: ["ALL"]
                       seccompProfile:
-                          type: RuntimeDefault
```

Comment:
Interesting... how did this ever work?

Reply:
YAML doesn't really care about the number of indents I guess, just that it's consistent within each block